test/Npgsql.Tests/Support/PgPostmasterMock.cs

```diff
@@ -21,12 +22,16 @@ class PgPostmasterMock : IAsyncDisposable
 
         readonly Socket _socket;
         readonly List<PgServerMock> _allServers = new List<PgServerMock>();
-        readonly Queue<PgServerMock> _pendingServers = new Queue<PgServerMock>();
-        readonly Queue<(int ProcessId, int Secret)> _pendingCancellationRequests
-            = new Queue<(int ProcessId, int Secret)>();
+        bool _acceptingClients;
         Task? _acceptClientsTask;
         int _processIdCounter;
 
+        ChannelWriter<ServerOrCancellationRequest> _pendingRequestsWriter { get; }
+        internal ChannelReader<ServerOrCancellationRequest> PendingRequestsReader { get; }
+
+        // internal ChannelReader<PgServerMock> PendingServers { get; }
+        // internal ChannelReader<(int ProcessId, int Secret)> PendingCancellationRequests { get; }
```

Comment:
Comments?

Reply:
Thanks yeah... But this is in #3253, will fix there.

Reply:
Cleaned this up in #3253